.github/workflows/performance.yml

```diff
@@ -152,7 +160,34 @@ jobs:
         run: npm run test:performance
 
       - name: Print performance tests results
-        run: "node ./tests/performance/results.js"
+        run: node ./tests/performance/results.js
+
+      - name: Check out target commit (target branch or previous commit)
+        run: |
+          if [[ -z "$TARGET_REF" ]]; then
+            git fetch -n origin $TARGET_SHA
+          else
+            git fetch -n origin $TARGET_REF
+          fi
+          git reset --hard $TARGET_SHA
+        env:
```

Comment:
Can we move these `env` variables to the workflow level?

`$TARGET_SHA` is used within another step further down in the workflow. Defining it once at the top would avoid having to repeat these and also gives a better spot to include any inline documentation we'd like it have explaining their values.